packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -0,0 +1,287 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+
+void main() {
```

Comment:
It would be useful to have a variable height items test where the item being dragged is as big as the list. Of course tests that scroll will be needed for that.